pkg/email/system_intake_unretire_lcid.go

```diff
@@ -0,0 +1,99 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"html/template"
+	"time"
+
+	"github.com/cms-enterprise/easi-app/pkg/models"
+)
+
+type systemIntakeUnretireLCIDEmailParameters struct {
+	LifecycleID              string
+	LifecycleExpiresAt       string
+	LifecycleIssuedAt        string
+	LifecycleScope           template.HTML
+	LifecycleCostBaseline    string
+	DecisionNextSteps        template.HTML
+	ITGovernanceInboxAddress string
+	AdditionalInfo           template.HTML
+}
+
+func (sie systemIntakeEmails) systemIntakeUnretireLCIDBody(
+	lifecycleID string,
+	lifecycleExpiresAt *time.Time,
+	lifecycleIssuedAt *time.Time,
+	lifecycleScope *models.HTML,
+	lifecycleCostBaseline string,
+	decisionNextSteps *models.HTML,
+	additionalInfo *models.HTML,
+) (string, error) {
+	var expiresAt string
+	if lifecycleExpiresAt != nil {
+		expiresAt = lifecycleExpiresAt.Format("01/02/2006")
+	}
+	var issuedAt string
+	if lifecycleIssuedAt != nil {
+		issuedAt = lifecycleIssuedAt.Format("01/02/2006")
+	}
+	data := systemIntakeUnretireLCIDEmailParameters{
+		LifecycleID:              lifecycleID,
+		LifecycleExpiresAt:       expiresAt,
+		LifecycleIssuedAt:        issuedAt,
+		LifecycleScope:           lifecycleScope.ToTemplate(),
+		LifecycleCostBaseline:    lifecycleCostBaseline,
+		DecisionNextSteps:        decisionNextSteps.ToTemplate(),
+		ITGovernanceInboxAddress: sie.client.config.GRTEmail.String(),
+		AdditionalInfo:           additionalInfo.ToTemplate(),
+	}
+
+	var b bytes.Buffer
+	if sie.client.templates.systemIntakeUnretireLCID == nil {
+		return "", errors.New("retire lcid template is nil")
+	}
```

Comment:
nit - can declare `b` below this if statement

```suggestion
	if sie.client.templates.systemIntakeUnretireLCID == nil {
		return "", errors.New("retire lcid template is nil")
	}
	var b bytes.Buffer
```

Reply:
bfdd1c1e1cd72c79a8df681d96d85eccd104f007